Wikimate.php

```diff
@@ -188,6 +188,34 @@ public function login($username, $password, $domain = null)
 		return true;
 	}
 
+	/**
+	 * Gets the user agent for API requests.
+	 *
+	 * @return  string  The default user agent, or the current one defined
+	 *                  by {@link Wikimate::setUserAgent()}
```

Comment:
A note ahead of the planned PHPDoc cleanup PR: this link isn't hyperlinked by my (possibly too old) phpdoc command. The login() link in the setUserAgent comments is, but no formatting variation attempts here will fix it in this comment. Letting this be for now.